harness/determined/common/api/authentication.py

```diff
@@ -218,6 +218,22 @@ def logout(
         pass
 
 
+def logout_all(master_address: Optional[str], cert: Optional[certs.Cert]) -> None:
```

Comment:
Nit: this should be `log_out_all` since it's a verb. On the other hand, `logout` is mashed together like this, so there's an argument that consistency is more important than grammar.